extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/NotFoundPageDisplayableAdditionalBuildItem.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.vertx.http.deployment.devmode;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+final public class NotFoundPageDisplayableAdditionalBuildItem extends MultiBuildItem {
```

Comment:
Could not come up with a better name. Suggestions are highly welcomed.

Reply:
Maybe there's an answer in your own code :) What do you think of `NotFoundPageDisplayableEndpointBuildItem`?

Reply:
Let's go with that. :-)